packages/kbn-optimizer/limits.yml

```diff
@@ -141,6 +141,7 @@ pageLoadAssetSize:
   triggersActionsUi: 135613
   uiActions: 35121
   uiActionsEnhanced: 38494
+  unifiedDocViewer: 66289
```

Comment:
I assume this was the limit before we optimized the page load size?

Reply:
Fixed in e23a04fe3adc4981b7f73da2962caae46e84e74e.